api/v1alpha1/dspipeline_types.go

```diff
@@ -90,26 +106,31 @@ type ArtifactScriptConfigMap struct {
 }
 
 type PersistenceAgent struct {
+	// Default: true
 	// +kubebuilder:default:=true
 	// +kubebuilder:validation:Optional
 	Deploy bool   `json:"deploy"`
```

Comment:
"Enable DS Pipelines Operator management of Persisence Agent. Setting Deploy to false disables operator reconciliation."